lib/src/core/verifier/sample/mod.rs

```diff
@@ -0,0 +1,125 @@
+use anyhow::{anyhow, Context, Result};
+use std::path::PathBuf;
+extern crate serde;
+use self::serde::{Deserialize, Serialize};
+use super::*;
+use crate::core::verifier::policy::opa;
+use crate::fs::*;
+use async_trait::async_trait;
+use serde_json::{json, Value};
+
+mod policy;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Ehd {
+    nonce: String,
+    public_key: String,
+}
+
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Tcb {
+    is_debuggable: bool,
+    cpusvn: u64,
+    svn: u64,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Quote {
+    is_debuggable: bool,
+    cpusvn: u64,
+    svn: u64,
+    report_data: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Policy {
```

Comment:
We're defining the same `Policy` struct for every verifier? It seems like this could be moved to the core file. The name policy is also a little bit confusing. This struct doesn't represent a policy, but the result of evaluating a policy.